qchem/tests/test_observable.py

```diff
@@ -9,37 +9,79 @@
 
 
 @pytest.mark.parametrize(
-    ("me_table", "init_term", "mapping", "terms_exp"),
+    ("matrix_elements", "init_term", "mapping", "terms_exp"),
     [
         (
-            np.array([[0.0, 0.0, 0.5], [1.0, 1.0, -0.5], [1.0, 0.0, 0.0]]),
+            [
+                np.array([[0.0, 0.0, 0.5], [1.0, 1.0, -0.5], [1.0, 0.0, 0.0]]),
+                np.array(
+                    [
+                        [0.0, 0.0, 0.0, 0.0, 0.5],
+                        [0.0, 1.0, 1.0, 0.0, -0.5],
+                        [1.0, 0.0, 0.0, 1.0, -1.0],
+                    ]
+                ),
+            ],
+            1 / 4,
+            "bravyi_KITAEV",
```

Comment:
Curious why the second word is capitalized?

Reply:
It is easier to identify different mappings and it also tests that the input is not case sensitive.